cypress/integration/route_spec.js

```diff
@@ -0,0 +1,98 @@
+describe('Sea route tests', function () {
+
+    before (function () {
+	cy.login();
+    });
+
+    beforeEach(function () {
+        // Session cookies will not be cleared before the NEXT test starts
+        cy.preserveSessionOnce();
+    });
+
+
+    it ("Create new route", function () {
+	cy.visit ("/ote/#/routes/");
+
+	cy.wait(500);
+
+	cy.get('tr').should('have.length', 1);
+
+	cy.contains(/Lisää uusi merireitti/).click();
+    });
+
+    it ("Add basic information", function () {
+	cy.get('[id*="Reitinnimi"]').type("Test route").should('have.value', "Test route");
+	cy.get('[id*="Palveluntuottaja"]').click();
+	cy.contains(/Testinen/).click();
+	cy.get('[id*="departure-point-name"]').type("Oulu").should('have.value', "Oulu");
+	cy.get('[id*="destination-point-name"]').type("Vaasa").should('have.value', "Vaasa");
+	cy.get('[id*="undefined--Voimassaalkaen"]').click();
```

Comment:
Do not include "undefined" in these queries. These tests will explode, if we fix the bug that produces these "undefined" values.

Reply:
Fixed

Reply:
Fixed